src/Classes/ServiceAPI/MyRadio_Timeslot.php

```diff
@@ -50,7 +51,7 @@ protected function __construct($timeslot_id)
         // Note that credits have different metadata timeranges to text
         // This is annoying, but needs to be this way.
         $result = self::$db->fetchOne(
-            'SELECT show_season_timeslot_id, show_season_id, start_time, duration, memberid, (
+            'SELECT show_season_timeslot_id, show_season_id, start_time, duration, memberid, playout::boolean::text, (
```

Comment:
Why the double-cast?

Reply:
can't remember. i'd guess its cause it didn't work for some strange reason